.github/workflows/unit-test.yml

```diff
@@ -88,13 +82,9 @@ jobs:
         npx lerna run build
 
     - name: Running the tests
-      env:
-        TZ: ${{ matrix.timezone }}
-        LANG: ${{ matrix.locale }}
 
       run: |
-        npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache" \
-          --testPathPattern="packages/date/"
+        npm run test-unit:date -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
```

Comment:
Can we just run this as part of the js unit tests job instead of having a separate job?

Reply:
Done!

Reply:
Potentially we could split the two lines into two "run" calls to have separate logs in the UI

Reply:
Done, much better! Thanks!